pyrk/para.sub

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
This file is full of commented out account information valid only for nuclear engineering at berkeley. While I understand the value of having utilities like this for running on the cluster, I think that a much better place for this information is in the documentation on the website.